src/management/src/management_frontend.hpp

```diff
@@ -63,30 +90,21 @@ std::string lookup_selection_state(
   return Policy::state_debug_string(state);
 }
 
-/**
- * @param models_node A ptree with a child object named `key` that
- * contains an array of VersionedModelId objects
- * objects.
- */
-std::vector<VersionedModelId> parse_candidate_models(
-    const ptree& models_node, ptree::key_type const& key) {
-  std::vector<VersionedModelId> models;
-  for (auto vm_json : models_node.get_child(key)) {
-    VersionedModelId cur_vm =
-        std::make_pair(vm_json.second.get<std::string>("model_name"),
-                       vm_json.second.get<int>("model_version"));
-    models.push_back(cur_vm);
-  }
-  return models;
-}
-
 void respond_http(std::string content, std::string message,
                   std::shared_ptr<HttpServer::Response> response) {
   *response << "HTTP/1.1 " << message
             << "\r\nContent-Length: " << content.length() << "\r\n\r\n"
             << content << "\n";
 }
 
+std::string json_error_msg(const std::string& exception_msg,
```

Comment:
Can we doc this to indicate that this message is used to return a user-facing error? This isn't clear by method name alone.